### PR TITLE
updating about.yml

### PR DESCRIPTION
**Pull request by porta-antiporta**
updating with current known information.  once final report is published, will submit another PR to update.

### CODE REVIEW

**Comment by porta-antiporta** [comment]
addresses #160

**Review by donjo — APPROVED** [review]
👍

File: .about.yml [hunk 1]
```diff
@@ -8,20 +8,20 @@ owner_type: project
...
     name: CC0
     url: https://github.com/18F/about_yml/blob/master/LICENSE.md
 tasks: https://github.com/18F/ffd-info-exchange/issues#boards
 contact:
-- url: mailto:michael.antiporta@gsa.gov #replace once we have a google group
-  text: Porta Antiporta #replace once we have a google group
+- url: mailto:ffd-info-exchange@gsa.gov #replace once we have a google group
+  text: Federal Front Door Information Exchange #replace once we have a google group
 team:
 - github: 18F/ffd-info-x
```
Comment (fureigh) [inline .about.yml:26]: `18f/ffd-info-x` doesn't appear to resolve. Let's list team members instead.

File: .about.yml [hunk 2]
```diff
@@ -8,20 +8,20 @@ owner_type: project
...
-- Postgres
 licenses:
   .about.yml:
     name: CC0
     url: https://github.com/18F/about_yml/blob/master/LICENSE.md
 tasks: https://github.com/18F/ffd-info-exchange/issues#boards
 contact:
-- url: mailto:michael.antiporta@gsa.gov #replace once we have a google group
-  text: Porta Antiporta #replace once we have a google group
+- url: mailto:ffd-info-exchange@gsa.gov #replace once we have a google group
```
Comment (fureigh) [inline .about.yml:23]: The comments can be deleted, as this is the email address for the Google group. :)

**Review by fureigh — COMMENTED** [review]
Already merged, but I can add a couple of quick commits today.

**Comment by fureigh** [comment]
Ah, @porta-antiporta already said he'd submit another PR to update! I'll leave this alone, then. :)

**Pull request merged**